src/layouts/errorPage/ErrorPage.js

```diff
@@ -0,0 +1,42 @@
+import Header from '../../components/header/Header';
+import Container from '../../components/container/Container';
+import Button from '../../components/button/Button';
+import redirectToTheHomepage from '../../utils/redirectToTheHomepage';
+
+class ErrorPage {
+  render() {
+    const container = document.createElement('section');
+
+    container.innerHTML = '';
+    container.classList.add('index-page__container');
+
+    const yodaImage = document.createElement('img');
```

Comment:
There could be `Image` component.

Reply:
I had issues with src when creating a new Image. Can we do it at the end of the project and replace all images created like above?